tdrs-frontend/src/components/EditProfile/EditProfile.jsx

```diff
@@ -14,25 +25,26 @@ function EditProfile() {
           First name
           <input
             className="usa-input"
-            id="first-name"
-            name="first-name"
+            id="firstName"
+            name="firstName"
```

Comment:
I am trying to stick to a pattern where any classes or names coming from USWDS follow the kebab case pattern and personal classes or names follow the snake case. Happy to change this as well.